task/backend/analytical_storage.go

```diff
@@ -34,7 +39,7 @@ const (
 // RunRecorder is a type which records runs into an influxdb
 // backed storage mechanism
 type RunRecorder interface {
-	Record(ctx context.Context, orgID platform.ID, org string, bucketID platform.ID, bucket string, run *taskmodel.Run) error
+	Record(ctx context.Context, bucketID platform.ID, bucket string, task *taskmodel.Task, run *taskmodel.Run) error
```

Comment:
Is this a change that requires a change to OpenAPI?

Reply:
I don't think so, I haven't found anywhere that this would be directly exposed as an API.